src/main/java/com/neuvector/Scanner.java

```diff
@@ -258,10 +237,27 @@ private static String runCMD(String[] cmdArgs, Logger log){
             // Read the output from the command
             String s = null;
             StringBuilder sb = new StringBuilder(String.join(" ", cmdArgs));
+            boolean catScanResultJsonOutput = false;
             while ((s = stdInput.readLine()) != null) {
+                if (outputCollector != null) {
+                    outputCollector.append(s);
```

Comment:
We log all output and also collect it into a StringBuilder that the caller can work with.